app/views/applications/show.html.erb

```diff
@@ -0,0 +1,49 @@
+<h1><%= @application.name %></h1>
+<h3>Address: <%= @application.address %></h3>
+<h3><i>Status: <%= @application.status %></i></h3>
+<p><%= @application.description %></p>
+
+<% @pets.each do |pet| %>
+  <%= link_to "#{pet.name}", "/pets/#{pet.id}" %></br>
+<% end %>
+
+<% if @application.status == "In Progress" %>
+  <div id="add-pet">
+    <h3> Add a Pet to this Application </h3>
+    <%= form_with url: "/applications/#{@application.id}", method: :get, data: {turbo: false} do |form| %>
+      <%= form.label :search_name, "Enter a name: " %>
+      <%= form.text_field :search_name %>
+      <%= form.submit "Submit" %>
+    <% end %>
+  </div>
+<% end %>
+
+<% if @pets_search_name.present? %>
```

Comment:
commenting here and in the controller because I'm not sure if your search process follows MVC - I think we might have to move all of this logic into a method or series of methods in the model and then just call it more simply in the controller and view